docs/en/Getting-Started.md

```diff
@@ -17,6 +17,7 @@ perform basic tasks with the data in Alluxio. During this guide, you will:
 * Perform basic tasks via Alluxio Shell
 * **[Bonus]** Mount a public Amazon S3 bucket in Alluxio
 * Shutdown Alluxio
+* Diagnose Alluxio startup issue
```

Comment:
This should be more of a "validate" step instead of a "diagnose" step? It would be good for people to run this before they start Alluxio, to avoid any issues when starting?